src/Resources/ResourceManager/Implementation/Policy/PSPolicyIdentity.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation.Policy
```

Comment:
This file should be renamed to `PsPolicyIdentity.cs` to match the casing of the class name

Reply:
Fixed in latest commit